### ISSUE TITLE
Title on ui.menu

### ISSUE THREAD

**Issue by Rmfjd1978** [h2oai/wave#1577]
I have to make a menu inside a header, but my menu needs to have some title

The solution is to have a parameter named label inside de ui.menu that is added to the chevron on a simple menu style.

the alternative solution that i have, is in one of the items of the header is a ui.inline with two items one ui.text and one ui.menu
```
await create_inline([await create_text(_content="App's"),
                                 await create_menu([await create_command(_name='#sss', _label='sss'),
                                                    await create_command(_name='#rrr', _label='rrr'),
                                                    await create_command(_name='#ttt', _label='ttt')])])
```

image of the pretended menu:
![image](https://user-images.githubusercontent.com/98091225/183475415-f9dc660a-ae57-459f-a670-f8d0be73bcb9.png)

In my alternative the clik event only acts where the chevron icon is clicked, if the menu had a label option the click event will act on the all object of the menu that includes the label and the chevron icon

**Comment by Pooja444** [comment]
@mturoci I would like to work on this issue, can it be assigned to me, please? Will this count towards Hacktoberfest, 2022?
Also, I will need to make changes to [this](https://github.com/h2oai/wave/blob/master/ui/src/menu.tsx) file, right?

**Comment by mturoci** [comment]
Hi @Pooja444 I will assign the issue to you. If you manage to make a valid PR that solves the problem correctly and the PR will get merged, it will count to Hacktoberfest submissions.

> Also, I will need to make changes to [this](https://github.com/h2oai/wave/blob/master/ui/src/menu.tsx) file, right?

Yes, this is the starting point, but it's not going to be the only file changed since you need to add a new attribute. See our contribution guidelines for further info.

Note that you are give 4 days before this issue becomes up for grabs again. Good luck!

**Comment by Pooja444** [comment]
@mturoci I have raised a PR for this issue. I tested it locally and have attached the test screenshots to the PR. Kindly review and let me know if I need to make any changes. Thanks for assigning me this issue!

**Comment by mturoci** [comment]
Closed by #1663
